### DISCUSSION TITLE
"update" in paperless shows [ERROR] in line 81: exit code 0: while executing command pip install -r requirements.txt &> /dev/null

### DISCUSSION THREAD

**Discussion by oswaldo-be** [tteck/Proxmox#3678] [Help Needed] [answered]
When I try to update my paperless instance to 2.12.0 via the update script it shows
✓ Stopped all Paperless-ngx Services
 / Updating to v2.12.0   
[ERROR] in line 81: exit code 0: while executing command pip install -r requirements.txt &> /dev/null
but 2.12.0 seems to be installed correctly.
When I manually try to install the
pip install -r requirements.txt manually it shows
-> error: externally-managed-environment

Is this even relevant?
Thank you very much

**Comment by tteck** [comment, marked as answer]
https://github.com/tteck/Proxmox/discussions/3300
